ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/NonValidatingPackageCommitter.scala

```diff
@@ -0,0 +1,24 @@
+package com.daml.ledger.participant.state.kvutils.committer
+
+import com.daml.ledger.participant.state.kvutils.committer.Committer.StepInfo
+import com.daml.metrics.Metrics
+
+/*
+ * This Package Committer do not run any validation but just prunes duplicated
+ *  packages from the entry.
+ *
+ * This should be used only by non distributed ledgers, like DAML-on-SQL,
+ * where the validation done in the API server is enough.
+ */
+private[committer] class NonValidatingPackageCommitter(metrics: Metrics)
+    extends PackageCommitter(metrics) {
+
+  override protected val committerName: String = "package_committer"
+
+  override protected val steps: Iterable[(StepInfo, Step)] =
+    List(
+      "deduplicate_submission" -> deduplicateSubmission,
```

Comment:
It makes me little bit nervous that the step lists are duplicated. If one needs to add a new step one needs to remember to add it to all three places. Since all of them are extra validation steps the order isn't as important, so could it be safer to have e.g. "extra_validation_steps" or some such that gets spliced into steps in PackageCommitter? What do you think?

Reply:
I share @dajmaki's concern.